contracts/vesting-lp/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "vesting-lp"
```

Comment:
why is it vesting-lp? what does lp mean here?

Reply:
Usually lp stands for liquidity provider, but to be honest I don't see anything related to market making in this contract.